presto-main/src/test/java/com/facebook/presto/server/TestHttpRequestSessionContext.java

```diff
@@ -161,7 +161,7 @@ public void testPreparedStatementsSpecialCharacters()
                         .put(PRESTO_LANGUAGE, "zh-TW")
                         .put(PRESTO_TIME_ZONE, "Asia/Taipei")
                         .put(PRESTO_CLIENT_INFO, "null")
-                        .put(PRESTO_PREPARED_STATEMENT, "query1=select * from tbl:ns")
+                        .put(PRESTO_PREPARED_STATEMENT, "query1=select * from \"tbl:ns\"")
```

Comment:
What is the reason for having this query with delimiter? If it's needed, we should have it generated using the parser so if in future we change the delimiter, this test won't break.

Reply:
Please the commit message [here](https://github.com/prestodb/presto/pull/17186/commits/f80abb112d62cdf61a34c804cada6ecffa5c6885)